editor/src/components/canvas/canvas-strategies/strategies/drag-to-insert-strategy.tsx

```diff
@@ -57,20 +57,20 @@ export function dragToInsertStrategy(
     controlsToRender: [
       // TODO the controlsToRender should instead use the controls of the actual canvas strategy -> to achieve that, this should be a function of the StrategyState here
       controlWithProps({
-        control: ParentOutlines,
-        props: {},
+        control: ImmediateParentOutlines,
+        props: { targets: [] }, // <<<- TODO feed it with targetParents
         key: 'parent-outlines-control',
         show: 'visible-only-while-active',
       }),
       controlWithProps({
-        control: ParentBounds,
-        props: {},
+        control: ParentBoundsForInsertion,
+        props: { targetParents: [] }, // <<<- TODO feed it with real props
```

Comment:
Are these TODOs for subsequent work?

Reply:
Yeah, there is a point about this in the Insertion doc, as this is only possible once insertion becomes a meta strategy